src/constructs/core/identity.ts

```diff
@@ -0,0 +1,5 @@
+export interface Identity {
+  stack: string;
+  stage: string;
+  app: string;
```

Comment:
Should `app` be optional?

Reply:
Hmm... I guess that might depend on the use case

Reply:
Good point. Let's get this in, we can always update it later if needed.